signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -101,6 +101,9 @@ public class RegistrationService {
     @Value("${mosip.signup.challenge.resend-attempt}")
     private int resendAttempts;
 
+    @Value("${mosip.signup.verification.attempt}")
```

Comment:
Rename the key to `mosip.signup.challenge.verification-attempt`

Reply:
renamed